client/src/services/sockets.js

```diff
@@ -19,14 +19,14 @@ class SocketManager {
   }
 
   unsubscribe(component) {
-    const findSubscriber = () => {
-      this.subscribers.forEach((subscriber, index) => {
-        if (subscriber.name === component) {
+    this.subscribers.splice(
+      this.subscribers.findIndex((item, index) => {
```

Comment:
Will it be better if we store subscribers inside an object? It would be easier to access and delete data.

Reply:
it works ok with an array

Reply:
just thinking on the possibility of having more subscribers, an array would not be efficient because findIndex is generally a loop.